virtwho/configure.py

```diff
@@ -1,4 +1,119 @@
-"""
-Used to manage virt-who configure file
+"""Used to manage virt-who configure file
```

Comment:
ummm, don't need this doc string because we have the details doc strings for each class, each function.
I added this doc string here, It's just a explanation :)

Reply:
Removed it :)